scripts/deploy/__init__.py

```diff
@@ -177,3 +180,28 @@ def run_deploy_twocrypto(chain: str, fee_receiver: str) -> None:
 @click.argument("coins", type=click.STRING)
 def run_deploy_twocrypto(chain: str, name: str, symbol: str, coins: str) -> None:
     deploy_pool(chain, name, symbol, coins.split(","))
+
+
+@deploy_commands.command("test_pools", short_help="deploy test tokens and pool on devnet")
+@click.argument("chain", type=click.STRING)
+def run_test_pools_deployment(chain: str) -> None:
+    assert "devnet" in chain, "Command only for devnets"
```

Comment:
we want to use the yaml file as much as possible here and not rely on whether the path has /devnet/ in it if possible.